backend/server.js

```diff
@@ -186,12 +186,84 @@ app.get("/user-activities", authenticateToken, async (req, res) => {
   try {
     const activities = await prisma.userActivity.findMany({
       where: { userId: req.user.id },
-      orderBy: { startDateTime: 'desc' },
-      take: 10 // Limit to 10 most recent activities
+      orderBy: { startDateTime: "desc" },
+      take: 10, // Limit to 10 most recent activities
```

Comment:
Might be more effort than it's worth, but you could look into adding a Javascript linter to your IDE so that these kinds of style differences get applied automatically on save (e.g. converting single quotes to double quotes or vice-versa depending on the linter).